vmware/vmware-cloud-director-ui-components
# Remove link from data exporter

## PR Type

What kind of change does this PR introduce?



## What does this change do?
Remove link from data exporter. We should not link to external sites.

## What manual testing did you do?
* Navigate to /dataExporter/example
* Clicked on Show Modal
* Clicked on the (i) icon next to the Sanitize Data checkbox
* Ensure that the link is no longer present

## Screenshots (if applicable)
<img width="1380" alt="image" src="https://user-images.githubusercontent.com/549331/232071670-91ea4a3e-cd14-4639-bcf5-e81d3a65ddcf.png">

## Does this PR introduce a breaking change?
